tests/ctst/world/Zenko.ts

```diff
@@ -1,15 +1,94 @@
-import { setWorldConstructor, World } from '@cucumber/cucumber';
+import { setWorldConstructor, World, IWorldOptions } from '@cucumber/cucumber';
 import axios, { AxiosRequestConfig, AxiosResponse, Method, AxiosRequestHeaders } from 'axios';
-import { aws4Interceptor } from 'aws4-axios'
-
-import { CacheHelper, cliModeObject, Constants, IAM, IAMUserPolicy, STS, SuperAdmin, Utils, } from 'cli-testing';
-import { extractPropertyFromResults } from "../common/utils";
+import { aws4Interceptor } from 'aws4-axios';
+import {
+    CacheHelper,
+    ClientOptions,
+    cliModeObject,
+    Constants,
+    IAM,
+    IAMUserPolicy,
+    STS,
+    SuperAdmin,
+    Utils,
+    AWSCliOptions,
+} from 'cli-testing';
+import { Credentials } from 'aws4-axios';
+import { extractPropertyFromResults } from '../common/utils';
 import qs = require('qs');
 
-interface AwsCliObjectParameters {
+export interface AwsCliObjectParameters {
     [key: string]: number | string | undefined | object;
 }
 
+export interface AWSVersionObject {
+    Key: string;
+    VersionId: string;
+}
+
+export interface NotificationDestination {
+    destinationName: string;
+    topic: string;
+    hosts: string;
+}
+export interface Saved {
+    notificationDestinations: NotificationDestination[],
+    notificationsPerDestination: { [key: string]: string[] },
+    bucketName: string,
+    notificationEventType: string,
+    objectNamePrefix: string,
+    filterType: string,
+    objectName: string,
+    objectNameSufix: string,
+    [key: string]: unknown
+}
+
+interface Parameters {
+    [key: string]: unknown;
+}
+
+export interface Result {
+    err: null;
```

Comment:
if err is `null` maybe we don't need it at all in the document, or maybe it can have another value?